### PR TITLE
Fix/cleaning up imports

### PR DESCRIPTION
**Pull request by lordbender**
I am looking at upgrading the UI from React 14 to React 16. As part of this, I am cleaning up some unused inputs and consolidating a few duplicate dependencies. There are no features added here, just removed a few unused dependencies.

There is minor formatting, but I tried not to pollute the PR with to much noise.

### CODE REVIEW

**Comment by lordbender** [comment]
@hunterford I went ahead and closed the upstream PR. I have another PR that will depend on this PR being merged. I am trying to clean up the items in the React code that are no logger supported in version greater that 15.4, like React.createClass and input refs (of type string), to name a few. It is also no longer supported to set state by direct assignment, so all of the this.state.varName = 'some value', will need to be moved to this.setState({varName: 'some value'}).

I am almost done with that effort, but it is quite large, so I was hoping not to muddy the waters with that PR until we get this one merged.

Thanks again for your time.

File: ui/gulpfile.babel.js [hunk 1]
```diff
@@ -144,20 +148,18 @@ gulp.task('watchTask', () => {
   });
 });
 
-gulp.task('deploy', function() {
-  return gulp.src(paths.distDeploy).pipe(ghPages());
-});
+gulp.task('deploy', () => gulp.src(paths.distDeploy).pipe(ghPages()));
 
 gulp.task('watch', cb => {
-  runSequence('clean', ['set-env', 'browserSync', 'watchTask', 'public', 'styles', 'fonts', 'lint', 'images'], cb);
```
Comment (lordbender) [inline ui/gulpfile.babel.js]: @hunterford this is a fix to a breaking change. Can we get this one merged?

**Comment by lordbender** [comment]
@hunterford Any thoughts on this merge request?

**Comment by lordbender** [comment]
@mstier-nflx or @hunterford : Can you please merge this PR?

**Comment by cyzhao** [comment]
@hunterford @mstier pls review this pr when you get a chance

**Pull request merged**